codegen/templates/chart/operator-deployment.yamltmpl

```diff
@@ -3,34 +3,28 @@ This template contains the core components for the Operator deployment.
 Expressions evaluating Helm Values use "{{" and "}}"
 Expressions evaluating SKv2 Config use "[[" and "]]"
 */]]
+
 [[- range $operator := $.Operators -]]
 [[- $operatorVar := (lower_camel $operator.Name) -]]
 
 [[/* custom values defined in codegen model */]]
-[[- $args := $operator.Deployment.Args -]]
 [[- $volumes := $operator.Deployment.Volumes -]]
-[[- $volumeMounts := $operator.Deployment.VolumeMounts -]]
-[[- $sidecars := $operator.Deployment.Sidecars -]]
 [[- $customPodLabels := $operator.Deployment.CustomPodLabels -]]
 [[- $customPodAnnotations := $operator.Deployment.CustomPodAnnotations -]]
 [[- $customDeploymentLabels := $operator.Deployment.CustomDeploymentLabels -]]
 [[- $customDeploymentAnnotations := $operator.Deployment.CustomDeploymentAnnotations -]]
 [[- $customServiceLabels := $operator.Service.CustomLabels -]]
-[[- $customServiceAnnotations := $operator.Service.CustomAnnotations ]]
+[[- $customServiceAnnotations := $operator.Service.CustomAnnotations -]]
```

Comment:
i'm not sure about using the `-]]` here to eliminate trailing whitespace. concerned we might get two strings concatenated to the same line